demos/signature/EVP_Signature_demo.c

```diff
@@ -0,0 +1,238 @@
+/*-
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* 
+ * An example that uses the EVP_MD*, EVP_DigestSign* and EVP_DigestVerify* 
+ * methods to calculate and verify a signature of two static buffers.
+ * Another way of generating a signature is to use EVP_PKEY_sign(), it uses
+ * a digest of message, does not hash the data to be signed, and therefore
+ * is normally used to sign digests.
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include "EVP_Signature_demo.h"
+#include <openssl/decoder.h>
+
+/*
+ * This demonstration will calculate and verify a signature of data using
+ * the soliloqy from Hamlet scene 1 act 3
```

Comment:
It's _soliloquy_

Reply:
Done in latest upload, thanks for your comments.